src/mobu/storage/nublado.py

```diff
@@ -591,10 +591,19 @@ async def auth_to_hub(self) -> None:
             Raised if no ``_xsrf`` cookie was set in the reply from the lab.
         """
         url = self._url_for("hub/home")
-        r = await self._client.get(url)
+        r = await self._client.get(url, follow_redirects=False)
+        # As with auth_to_lab, manually extract from cookies at each
+        # redirection, because httpx doesn't do that if following redirects
+        # automatically.
+        while r.is_redirect:
+            xsrf = self._extract_xsrf(r)
+            if xsrf and xsrf != self._lab_xsrf:
```

Comment:
Why do we need to check this against the `_lab_xsrf`? My initial understanding is that we want to change the `_hub_sxrf` if we get a different one back in any of the responses.

I'm actually sort of confused as to why we're comparing the `_lab_xsrf` to the `_hub_xsrf` in [auth_to_lab](https://github.com/lsst-sqre/mobu/pull/365/files#diff-7f1deca6c43fb3b286f3e4b110994958679c83421ade4a82b880ece90cee3927R638) also, but that's not being changed here...

Reply:
Do we even need to check if the token from the response cookie is different than the one we're storing, or can we just say "if we get an xsrf cookie in a response, store it"?

Reply:
```python
if xsrf:
    self._hub_xsrf = xsrf
```
?

Reply:
Hm.  I misunderstood the intention.  I want to check if the token is different than the one we store to cut down on the number of times we log that we're setting the token, so that I can also understand the cadence with which we get genuinely new ones.

Reply:
Ahh gotcha.  Yeah, with the logging it makes sense to keep the check.  My main question was about comparing the hub token to the lab token and vice vera, and it looks like that was a bug originally and is fixed now.